network-api/networkapi/wagtailpages/pagemodels/buyersguide/products.py

```diff
@@ -246,15 +247,28 @@ def get_votes(self):
         votes = [int(x) for x in self.vote_bins.split(",")]
         return votes
 
-    def get_most_voted(self):
+    def get_vote_average(self):
         votes = self.get_votes()
         vote_breakdown = {k: v for (k, v) in enumerate(votes)}
-        highest = max(vote_breakdown, key=vote_breakdown.get)
-        label = self.get_vote_labels()[highest]
+        total_votes = sum(vote_breakdown.values())
```

Comment:
I wonder if we could return even earlier for 0 vote scenario?

Something like 


```py
def get_vote_average(self):
        votes = self.get_votes()
        total_votes = sum(votes)

        """If there's no votes, let the user know (also protects from division by zero errors)"""
        if total_votes == 0:
            ...
            return {
                ...
            }
        
        vote_breakdown = {k: v for (k, v) in enumerate(votes)}
        weighted_votes = 0
        ...
```

Reply:
Sure, might as well!